server/huntflow_reloaded/handler.py

```diff
@@ -207,23 +227,49 @@ def _process_request(self):
 
         options = {
             "created": today,
-            "type": event.get('type'),
+            "type": self.event.get('type'),
             "candidate": _id,
             "start": interview_start,
             "end": interview_end
         }
 
         interview = await models.Interview.create(**options)
 
-        message = {
+        self.event_type = 'schedule_interview' # pylint: disable=attribute-defined-outside-init
+
+        # pylint: disable=attribute-defined-outside-init
+        self.message = {
             "type": message_type,
-            "first_name": first_name,
-            "last_name": last_name,
+            "first_name": self.basic_attrs['first_name'],
+            "last_name": self.basic_attrs['last_name'],
             "start": start,
         }
 
-        await self._scheduler.create_event(message, interview)
-        self._scheduler.publish_now(message)
+        # pylint: disable=attribute-defined-outside-init
+        self.context = {"message": self.message,
+                        "interview": interview}
+
+    async def handle_employment_date(self):
+        """Handles the setting of the first working day. """
+
+        try:
+            employment_date = self.event['employment_date']
+        except KeyError:
+            raise IncompleteRequest
+
+        self.event_type = 'remove_candidate' # pylint: disable=attribute-defined-outside-init
+
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
This line ignore all warnings in scope. You should use it on the line which pylint is warn.